pipeline-modules/deployment-service/gcp/private-helm.yaml

```diff
@@ -114,10 +105,14 @@ template: |
       kubectl create ns {{namespace}}
       {% endif %}
       cd {{source_code_path}}/{{helm_chart_path}} ;
+      {%- if helm_values_path != '' %}
+      cp {{source_code_path}}/{{helm_values_path}} values_presub.yml
+      cat values_presub.yml | envsubst > {{source_code_path}}/{{helm_values_path}}
```

Comment:
@dshelley66 I assume that this line add environment variable & substitution in values.yaml file right, have you tested this ,is this working fine right

Reply:
Yes and Yes. I have defintely tested this (with my json-extract example). We should definitely do more testing...